app/models/vm_resize_request.rb

```diff
@@ -0,0 +1,26 @@
+class VmResizeRequest < MiqRequest
```

Comment:
My suggestion is to use "VmCloudReconfigure" for this work.  So this class name would be `VmCloudReconfigureRequest` and the task would be `VmCloudReconfigureTask`.